verilog/formatting/tree_unwrapper_test.cc

```diff
@@ -773,6 +773,39 @@ const TreeUnwrapperTestData kUnwrapModuleTestCases[] = {
         L(0, {"endmodule"}),
     },
 
+    {
+        "module with static variable",
+        "module static_variable;\n"
+        "initial begin\n"
+        "  static int a = 0;\n"
+        "end\n"
+        "endmodule",
+        ModuleHeader(0, L(0, {"module", "static_variable", ";"})),
+        ModuleItemList(1, L(1, {"initial", "begin"}),
+                       StatementList(2, N(2, L(2, {"static"}), L(2, {"int"}),
+                                             L(2, {"a", "=", "0", ";"}))),
+                       L(1, {"end"})),
+        L(0, {"endmodule"}),
+    },
+
+    {
+        "module with static and automatic variables",
+        "module static_automatic;\n"
+        "initial begin\n"
+        "  static int a = 0;\n"
+        "  automatic byte b = 1;\n"
+        "end\n"
+        "endmodule",
+        ModuleHeader(0, L(0, {"module", "static_automatic", ";"})),
+        ModuleItemList(1, L(1, {"initial", "begin"}),
+                       StatementList(2, N(2, L(2, {"static"}), L(2, {"int"}),
+                                             L(2, {"a", "=", "0", ";"})),
+                                        N(2, L(2, {"automatic"}), L(2, {"byte"}),
+                                             L(2, {"b", "=", "1", ";"}))),
+                       L(1, {"end"})),
+        L(0, {"endmodule"}),
+    },
+
```

Comment:
Can you add a test case that declares more than one variable in one declaration?  such as `static int a=0, b=0;`